README.md

```diff
@@ -2,116 +2,221 @@
 
 ## Overview
 
-Web components which convert any web page into a shopping cart.
+Web components to convert any web page into a shopping cart.
 
-This includes components to:-
- - Dynamically update price and qty for listed items. 
+Web Components to:-
+ - Dynamically update price and qty for items. 
  - Add items to a cart and manage cart.
  - Authenticate and checkout.
  
-Once customers are authenticated, price / qty shown and the types of payment / shipping methods available may change.
-This functionality is usually specific to B2B e-commerce.
+Authenticated customer may see price, qty, payment methods and shipping methods specific to them.
+This functionality is usually associated with B2B e-commerce.
 
-_For a website to use these components, it should include a single `.js` file and the appropriate components in the pages HTML._
+The components are designed to be used by different audiences:-
+- Web developers with a basic understanding of HTML to quickly add e-commerce functionality to their site.
+- Javascript software engineers who wish to integrate into different applications or have greater flexibility to design custom carts.    
 
-## Web Components
 
-Required web components
+Requirements for [SLC](https://herman.bearblog.dev/mvp-vs-slc/)
```

Comment:
This doesn't belong in the README, it's just a side note, and not of much consequence